packages/components/tests/dummy/app/templates/components/page-header.hbs

```diff
@@ -0,0 +1,361 @@
+{{!
+Copyright (c) HashiCorp, Inc.
+SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "PageHeader Component"}}
+
+<Shw::Text::H1>PageHeader</Shw::Text::H1>
+
+<section data-test-percy>
+
+  <Shw::Text::H2>Basic examples</Shw::Text::H2>
+
+  <Shw::Flex @direction="column" as |SF|>
+
+    <SF.Item @label="Only title">
+
+      <Hds::PageHeader as |PH|>
```

Comment:
Super nit: If content is indented it doesn't also need blank lines above and below the indented portion.